modules/publicize/ui.php

```diff
@@ -266,6 +316,54 @@ function admin_page() {
 
 	}
 
+	function display_connection_error() {
```

Comment:
Same question: Does `display_connection_error()` need to go into ui.php? That file is shared between WP.com and Jetpack.